frontend/src/app/pages/exercises/exercise/shared/exercise-map/utility/ol-geometry-helpers.ts

```diff
@@ -0,0 +1,118 @@
+import { Position } from 'digital-fuesim-manv-shared';
+import type { Size, UUID } from 'digital-fuesim-manv-shared';
+import { Feature } from 'ol';
+import type { Coordinate } from 'ol/coordinate';
+import type { Geometry } from 'ol/geom';
+import { Point, Polygon } from 'ol/geom';
+import type { WithPosition } from '../../utility/types/with-position';
+
+export interface PositionableElement {
+    readonly id: UUID;
+    readonly position: Position;
+}
+
+export type ResizableElement = PositionableElement & {
+    size: Size;
+};
+
+export type GeometryWithCoorindates = Geometry & {
```

Comment:
```suggestion
export type GeometryWithCoordinates = Geometry & {
```

Reply:
Good catch, I've fixed it using find and replace :)